<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportInventory</class>
 <widget class="QMainWindow" name="ImportInventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Inventory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="mainGridLayout" columnstretch="5,0,0">
      <item row="2" column="0">
       <widget class="QCheckBox" name="splitInventoryCheckBox">
        <property name="text">
         <string>Split inventory</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="featureLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="importFeatureLabel">
        <property name="text">
         <string>Import feature</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLineEdit" name="testDatasetLineEdit"/>
      </item>
      <item row="7" column="2">
       <widget class="QToolButton" name="trainingDatasetToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="trainingDatasetLabel">
        <property name="text">
         <string>Training dataset</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="testDatasetLabel">
        <property name="text">
         <string>Test dataset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="2">
       <widget class="QToolButton" name="testDatasetToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="featureToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLineEdit" name="trainingDatasetLineEdit"/>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="applyPushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="sizeValueLineEdit">
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QSlider" name="sizeValueHorizontalSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="sliderPosition">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sizeOfTrainingDatatsetLabel">
        <property name="text">
         <string>Size of the training dataset in %:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="RandomSeedlineEdit">
        <property name="toolTip">
         <string>Leave empty to use random seed</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="randomseedlabel">
        <property name="text">
         <string>Seed to initialize random</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>featureLineEdit</tabstop>
  <tabstop>featureToolButton</tabstop>
  <tabstop>splitInventoryCheckBox</tabstop>
  <tabstop>sizeValueLineEdit</tabstop>
  <tabstop>RandomSeedlineEdit</tabstop>
  <tabstop>sizeValueHorizontalSlider</tabstop>
  <tabstop>trainingDatasetLineEdit</tabstop>
  <tabstop>trainingDatasetToolButton</tabstop>
  <tabstop>testDatasetLineEdit</tabstop>
  <tabstop>testDatasetToolButton</tabstop>
  <tabstop>applyPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
